### ISSUE TITLE
[Bug] [Import-DbaCsv] Failure | The property 'Encoding' cannot be found on this object. Verify that the property exists and can be set.

### ISSUE THREAD

**Issue by DanielEms2** [sqlcollaborative/dbatools#5634]
### Environmental information

<!--
Run below command, paste results below:
& {"``````";"#### PowerShell version:`n$($PSVersionTable | Out-String)"; "`n#### dbatools Module version:`n$(gmo dbatools -List | select name, path, version | fl -force | Out-String)";"``````"} | clip
-->
```
#### PowerShell version:

Name                           Value                                                                                                                                                                                                                                                                                                                      
----                           -----                                                                                                                                                                                                                                                                                                                      
PSVersion                      5.1.17134.765                                                                                                                                                                                                                                                                                                              
PSEdition                      Desktop                                                                                                                                                                                                                                                                                                                    
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                                                                                                                                                    
BuildVersion                   10.0.17134.765                                                                                                                                                                                                                                                                                                             
CLRVersion                     4.0.30319.42000                                                                                                                                                                                                                                                                                                            
WSManStackVersion              3.0                                                                                                                                                                                                                                                                                                                        
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                                                                                                                                        
SerializationVersion           1.1.0.1                                                                                                                                                                                                                                                                                                                    


#### dbatools Module version:

Name    : dbatools
Path    : C:\Program Files\WindowsPowerShell\Modules\dbatools\0.9.832\dbatools.psd1
Version : 0.9.832
```


#### SQL Server: 
<!-- Paste output of `SELECT @@VERSION` -->
```sql
Microsoft SQL Server 2012 (SP3) (KB3072779) - 11.0.6020.0 (X64) 
	Oct 20 2015 15:36:27 
	Copyright (c) Microsoft Corporation
	Developer Edition (64-bit) on Windows NT 6.1 <X64> (Build 7601: Service Pack 1) (Hypervisor)
```

### Report

<!--
Things to consider:
- Errors received are not related to permissions?
- If this refers to Copy-DbaDatabase can you replace the problem with Backup-DbaDatabase and Restore-DbaDatabase?
- Copy-DbaDatabase will not work in every environment and every situation. Instead, we try to ensure Backup & Restore work in your environment.
--> 

#### Errors Received

<!-- 
Run this command and paste below:
& {"``````";$error[0] | select *;"``````"} | clip
-->
```
writeErrorStream      : True
PSMessageDetails      : 
Exception             : System.Exception: The property 'Encoding' cannot be found on this object. Verify that the property exists and can be set. ---> System.Management.Automation.RuntimeException: The property 'Encoding' cannot be found on this object. Verify that the property exists and can be set.
                           at System.Management.Automation.ExceptionHandlingOps.CheckActionPreference(FunctionContext funcContext, Exception exception)
                           at System.Management.Automation.Interpreter.ActionCallInstruction`2.Run(InterpretedFrame frame)
                           at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
                           at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
                           --- End of inner exception stack trace ---
TargetObject          : 
CategoryInfo          : InvalidOperation: (:) [Write-Error], Exception
FullyQualifiedErrorId : dbatools_Import-DbaCsv,Stop-Function
ErrorDetails          : The property 'Encoding' cannot be found on this object. Verify that the property exists and can be set.
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at Stop-Function, C:\Program Files\WindowsPowerShell\Modules\dbatools\0.9.832\allcommands.ps1: line 110288
                        at Import-DbaCsv<Process>, C:\Program Files\WindowsPowerShell\Modules\dbatools\0.9.832\allcommands.ps1: line 56618
                        at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {0, 1}

```
#### Steps to Reproduce

<!--
If you have confirmed this issue can be reproduced, please provide the exact steps (T-SQL, PowerShell, ext)
-->
Create the test table using this sql
[create_table_sql.txt](https://github.com/sqlcollaborative/dbatools/files/3232602/create_table_sql.txt)
Run this command on the attached file (or probably any file):
[test.txt](https://github.com/sqlcollaborative/dbatools/files/3232582/test.txt)
```powershell
Import-DbaCsv -Path "C:\Temp\test.txt" -SqlInstance SQLInstance -Database SQLDatabase -Table "test" -Encoding ([System.Text.Encoding]::UTF8)
```
#### Expected Behavior

<!--
What did you expect to happen?
-->
I expected the file to be imported into SQL Server.  It works without the -Encoding parameter.


#### Actual Behavior

<!--
What happened?
-->
```powershell
WARNING: [09:04:49][Import-DbaCsv] Failure | The property 'Encoding' cannot be found on this object. Verify that the property exists and can be set.
```

**Comment by potatoqualitee** [comment]
Thanks for the detailed bug report, ill fix that and having to do  ([System.Text.Encoding]::UTF8) within 20 days 👍

**Comment by potatoqualitee** [comment]
will be resolved in 1.0, which will be released in 20 days 💯
